proxy/logging/LogField.cc

```diff
@@ -772,7 +772,9 @@ LogFieldList::marshal_len(LogAccess *lad)
   int bytes = 0;
   for (LogField *f = first(); f; f = next(f)) {
     if (f->type() != LogField::sINT) {
-      bytes += f->marshal_len(lad);
+      const int len = f->marshal_len(lad);
```

Comment:
I am not seeing that in the code on master `round_strlen(m_client_req_unmapped_url_host_len + 1);`.  What line are you referring to?